quest/util/misc.py

```diff
@@ -88,17 +100,35 @@ def bbox2poly(x1, y1, x2, y2, reverse_order=False, as_geojson=False, as_shapely=
 
 
 def build_smtk(smtk_subdir, smtk_filename, **kwargs):
-    """build smtk file from template and kwargs and stream as string.
+    """Build smtk file from the template, kwargs, and stream as a string.
+
+    Args:
+        smtk_subdir (string):
+        smtk_filename (string):
+        kwargs (kwargs):
+
+    Returns:
 
     """
     env = Environment(loader=FileSystemLoader(os.path.join(get_pkg_data_path('smtk'), smtk_subdir)))
     return env.get_template(smtk_filename).render(**kwargs)
 
 
 def classify_uris(uris, grouped=True, as_dataframe=True, require_same_type=False, exclude=None):
-    """convert list of uris into a pandas dataframe.
+    """Converts a list of uris into a pandas dataframe.
+
+    Notes:
+        Classified by resource type.
 
-    classified by resource type
+    Args:
+        uris (?):
```

Comment:
+        uris (list or string): List of Quest uris to classify into the following types: 'collections', 'services', 'publishers', 'datasets', or 'features'.
+        grouped (bool): If True returns Pandas GroupBy object (see: https://pandas.pydata.org/pandas-docs/stable/groupby.html)
+        as_dataframe (bool): If True returns a Pandas DataFrame
+        require_same_type (bool): If True raises a `ValueError` if uris of more than one type are passed in.
+        exclude (list or string): List of uri types to not allow. If a uri of an excluded type is passed in then a `ValueError` will be raised.